easy_thumbnails/alias.py

```diff
@@ -111,14 +111,10 @@ def _coerce_target(self, target):
         """
         if not target or isinstance(target, str):
             return target
-        if not hasattr(target, 'instance'):
+        if not hasattr(target, "instance"):
             return None
         model = target.instance.__class__
-        return '%s.%s.%s' % (
-            model._meta.app_label,
-            model.__name__,
-            target.field.name,
-        )
+        return "%s.%s.%s" % (model._meta.app_label, model.__name__, target.field.name,)
```

Comment:
As far as I remember @SmileyChris  [prefers](https://github.com/SmileyChris/easy-thumbnails/pull/257#discussion_r6099584) the `"{}".format(foo)` syntax over `"%s" % foo`.